From f665c585ee9ea7beec8de94a4d402190225ccf23 Mon Sep 17 00:00:00 2001
From: "A. Maitland Bottoms" <bottoms@debian.org>
Date: Wed, 20 Oct 2021 10:11:16 -0500
Subject: [PATCH] responder: Fix printw function arguments

This is necessary when compiling with -Werror=format-security as Debian
package builds do.

Thanks to mait for the fix!
---
 host/utils/latency/lib/Responder.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/host/utils/latency/lib/Responder.cpp b/host/utils/latency/lib/Responder.cpp
index 65c3e2941..5c8303d25 100644
--- a/host/utils/latency/lib/Responder.cpp
+++ b/host/utils/latency/lib/Responder.cpp
@@ -56,7 +56,7 @@ void Responder::register_stop_signal_handler()
 // For ncurses. Print everything in stream to screen
 void Responder::FLUSH_SCREEN()
 {
-    printw(_ss.str().c_str());
+    printw("%s", _ss.str().c_str());
     refresh();
     _ss.str("");
 }
